Java/serialization-json/src/test/java/com/amazon/randomcutforest/serialize/RandomCutForestSerDeTests.java

```diff
@@ -31,7 +31,7 @@
 
     private static final int dimensions = 4;
     private static final int numberOfTrees = 30;
-    private static final int sampleSize = 128;
+    private static final int sampleSize = 28000;
```

Comment:
Did you mean to leave this in? It's not super important in the long run since we plan to remove this class.

Reply:
Yes, this is a test that checks that the state variables are handled for the larger NodeStore/LeafStore (hence the 28000). This test will likely move to somewhere else - but for now this seemed to be a good place to have it.